int64 Quality
sensor_msgs/Range sensor_values